docker/docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  app:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    ports:
+      - '80:3000'
```

Comment:
Propose adding ability to specify the port
```
      - "${PORT:-80}:3000"
```

Reply:
Probably ok to have port 80 and users to edit the compose file directly if need be.